modules/imaging_browser/jsx/ImagingBrowserSDT.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Need to override static data table in order to maintain sorting order
+ * from the menu filter in imaging viewer
+ */
+/* global StaticDataTable */
+class ImagingBrowserSDT extends StaticDataTable {
+  componentDidUpdate() {
+    if (jQuery.fn.DynamicTable) {
+      if (this.props.freezeColumn) {
+        $("#dynamictable").DynamicTable({
+          freezeColumn: this.props.freezeColumn
+        });
+      } else {
+        $("#dynamictable").DynamicTable();
+      }
+    }
+
+    // Start Imaging Browser 17.1 Quickfix
+    var index = this.getSortedRows();
```

Comment:
in StaticDataTable:
https://github.com/aces/Loris/blob/17.1-dev/jsx/StaticDataTable.js#L183